utils/box-symbol.stanza

```diff
@@ -244,70 +253,14 @@ public defn make-box-symbol (banks:Seqable<BoxBank>) :
 ;=======================================================<doc>
 
 public defn make-box-symbol () :
-  ;Sort the given pins by their .pin-properties-row-index field.
-  defn sort-by-row-index (pins:Tuple<JITXObject>) -> Tuple<JITXObject> :
-    ;Compare pin[i0] versus pin[i1].
-    ;Indices are used, instead of the pin directly, in order to perform a
-    ;stable sort.
-    ;All pins without a .pin-properties-row-index are put in the beginning.
-    defn compare-index (i0:Int, i1:Int) :
-      val ai = value?(property?(pins[i0].pin-properties-row-index))
-      val bi = value?(property?(pins[i1].pin-properties-row-index))
-      match(ai, bi) :
```

Comment:
These functions are shared by `assign-symbols`, so the helper functions / checkers are moved to ~ line 346.